tests/SUMMARY_EFF_FAC.DATA

```diff
@@ -160,7 +178,7 @@ COMPDAT
    W_1 0 0 1 1 /   -- Active index: 0
    W_2 0 0 1 1 /   -- Active index: 1
    W_2 0 0 2 2 /   -- Active index: 101
-   W_3 0 0 1 1 /   -- Active index: 2
+   W_3 3 1 1 1 /   -- Active index: 2
```

Comment:
> Can the `-- Active index: 2` still be correct?

Actually, yes.  It's correct now (assuming we count the active indices from zero), whereas it was wrong before.  Since `0 0` picks this information up from `WELSPECS`, it would be treated as [`2 1`](https://github.com/OPM/opm-common/blob/a48c3829d739d4ae0a774bc3c75ac89a73eb15f4/tests/SUMMARY_EFF_FAC.DATA#L136) before which would be active index 1.